### DISCUSSION TITLE
Performance optimization

### DISCUSSION THREAD

**Discussion by anarion80** [matt8707/hass-config#196] [General]
Continuing from the closed issue https://github.com/matt8707/hass-config/issues/165, is there any update/progress on your new, crazy-fast, non-lovelace dashboard @matt8707 ? :)

**Comment by matt8707** [comment]
Yeah, nearing a pre-release. However, I'm swamped with studies, so it'll drop when it's ready

**Reply by anarion80** [reply]
Excellent! Looking forward to the release.

**Reply by Suschibomber** [reply]
I'm also really excited.
